homeassistant/components/konnected/panel.py

```diff
@@ -0,0 +1,310 @@
+"""Support for Konnected devices."""
+import logging
+
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    ATTR_STATE,
+    CONF_ACCESS_TOKEN,
+    CONF_BINARY_SENSORS,
+    CONF_DEVICES,
+    CONF_HOST,
+    CONF_ID,
+    CONF_NAME,
+    CONF_PORT,
+    CONF_SENSORS,
+    CONF_SWITCHES,
+    CONF_TYPE,
+    CONF_ZONE,
+)
+from homeassistant.core import callback
+from homeassistant.helpers import aiohttp_client, device_registry as dr
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+
+from .errors import CannotConnect
+from .const import (
+    CONF_ACTIVATION,
+    CONF_API_HOST,
+    CONF_BLINK,
+    CONF_DHT_SENSORS,
+    CONF_DISCOVERY,
+    CONF_DS18B20_SENSORS,
+    CONF_INVERSE,
+    CONF_MOMENTARY,
+    CONF_PAUSE,
+    CONF_POLL_INTERVAL,
+    CONF_REPEAT,
+    DOMAIN,
+    ENDPOINT_ROOT,
+    SIGNAL_SENSOR_UPDATE,
+    STATE_LOW,
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class AlarmPanel:
```

Comment:
cool, so is this also an entity in Home Assistant or just a class used for config?

Reply:
Just a class used to manage config of the panel.   It's basically a combo of the discovery and config classes that used to reside in init.